PIL/PalmImagePlugin.py

```diff
@@ -83,10 +83,10 @@ def build_prototype_image():
     image = Image.new("L", (1, len(_Palm8BitColormapValues),))
     image.putdata(list(range(len(_Palm8BitColormapValues))))
     palettedata = ()
-    for i in range(len(_Palm8BitColormapValues)):
-        palettedata = palettedata + _Palm8BitColormapValues[i]
+    for colormapValue in _Palm8BitColormapValues:
+        palettedata += colormapValue
     for i in range(256 - len(_Palm8BitColormapValues)):
-        palettedata = palettedata + (0, 0, 0)
+        palettedata += (0, 0, 0)
```

Comment:
This could probably be: 
`palettedata += (0,0,0)*(256 - len(_Palm8BitColormapValues)`

And the above loop could probably be:
`palettedata = sum(_Palm8BitColormapValues, ())`, but I'm not sure if that's clear or if it's just code golf. 

(though, if sum were renamed append...  ` def append(seq_of_tuples): return sum(seq_of_tuples,())`